x-pack/plugins/threat_intelligence/public/modules/indicators/indicators_page.tsx

```diff
@@ -54,8 +55,14 @@ export const IndicatorsPage: VFC = () => {
           onSubmitDateRange={handleSubmitTimeRange}
         />
       </FiltersGlobal>
-      <IndicatorsBarChartWrapper timeRange={timeRange} indexPattern={indexPattern} />
-      <IndicatorsTable {...indicators} browserFields={browserFields} indexPattern={indexPattern} />
+      <IndicatorsFilters filterManager={filterManager}>
```

Comment:
this `IndicatorsFiltersContext` component allows us to share `filterManager` (received from the `KQLBar`) between all the components inside the `IndicatorsTable` (like `CellActions` and `IndicatorsFlyoutTable`)